content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Network configuration objects
+description: Describes Istio's network configuration objects and their functionality.
+weight: 30
+keywords: [network, virtual service, gateway, ingress, egress, service entry]
+aliases:
+---
+
+## The Istio network configuration objects and your application
```

Comment:
Please don't start with a title. The page as a whole already has a title rendered by the site infra.